test/tools/CodeCoverageAutomation/Start-CodeCoverageRun.ps1

```diff
@@ -14,6 +16,30 @@ function Write-LogPassThru
     Add-Content -Path $Path -Value $Message -PassThru -Force
 }
 
+function Write-BashInvokerScript
+{
+    param($path)
+
+    $scriptContent =
+    @'
+    @echo off
+    setlocal
+
+    if not exist "%~dpn0.sh" echo Script "%~dpn0.sh" not found & exit 2
+
+    set _CYGBIN=C:\cygwin64\bin
+    if not exist "%_CYGBIN%" echo Couldn't find Cygwin at "%_CYGBIN%" & exit 3
+
+    :: Resolve ___.sh to /cygdrive based *nix path and store in %_CYGSCRIPT%
+    for /f "delims=" %%A in ('%_CYGBIN%\cygpath.exe "%~dpn0.sh"') do set _CYGSCRIPT=%%A
+
+    :: Throw away temporary env vars and invoke script, passing any args that were passed to us
+    endlocal & %_CYGBIN%\bash --login "%_CYGSCRIPT%" %*
+'@
+
+    $scriptContent | Out-File $path -Force
```

Comment:
if it's not needed, it's fine

---
In reply to: [97681882](https://github.com/PowerShell/PowerShell/pull/3037#discussion_r97681882) [](ancestors = 97681882)